src/order-history/services.js

```diff
@@ -0,0 +1,40 @@
+import pick from 'lodash.pick';
+
+let config = {
+  ACCOUNTS_API_BASE_URL: null,
+  LMS_BASE_URL: null,
+};
+
+let apiClient = null; // eslint-disable-line no-unused-vars
+
+function validateConfiguration(newConfig) {
+  Object.keys(config).forEach((key) => {
+    if (newConfig[key] === undefined) {
+      throw new Error(`Service configuration error: ${key} is required.`);
+    }
+  });
+}
+
+export function configureServices({ configuration: newConfig, apiClient: newApiClient }) {
```

Comment:
I'd just simplify this down to two vanilla parameters... it feels a bit weird to package them into an object on one side and then just unpack the object on this side.